src/OrchardCore/OrchardCore.ContentManagement.Abstractions/VersionOptions.cs

```diff
@@ -0,0 +1,35 @@
+namespace OrchardCore.ContentManagement;
+
+public class VersionOptions
```

Comment:
@sebastienros should we change this type to `record struct` instead of class?

Reply:
That would work I thinkg, doesn't matter much if it's easy you can do it

Reply:
done.